Tests/Amr/Advection_AmrCore/Source/AmrCoreAdv.cpp

```diff
@@ -161,16 +172,20 @@ AmrCoreAdv::InitData ()
         InitFromScratch(time);
         AverageDown();
 
+#ifdef AMREX_PARTICLES
+    if (do_tracers) {
+        init_particles();
+    }
```

Comment:
Alignment with code above is a bit off:
```suggestion
        if (do_tracers) {
            init_particles();
        }
```